From c30dffa142f75716ea36b54e17da40d5e4e20bc4 Mon Sep 17 00:00:00 2001
From: Soren Brinkmann <soren.brinkmann@xilinx.com>
Date: Mon, 3 Dec 2012 14:10:44 -0800
Subject: [PATCH] mtd: xilinx_nandps: Add const qualifier to xnandps_commands

The array is never modified and can be declared const.

Signed-off-by: Soren Brinkmann <soren.brinkmann@xilinx.com>
Acked-by: John Linn <john.linn@xilinx.com>
Acked-by: Michal Simek <michal.simek@xilinx.com>
---
 drivers/mtd/nand/xilinx_nandps.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/mtd/nand/xilinx_nandps.c b/drivers/mtd/nand/xilinx_nandps.c
index 24b8dae..f16602e 100644
--- a/drivers/mtd/nand/xilinx_nandps.c
+++ b/drivers/mtd/nand/xilinx_nandps.c
@@ -174,7 +174,7 @@ struct xnandps_info {
 /*
  * The NAND flash operations command format
  */
-static struct xnandps_command_format xnandps_commands[] = {
+static const struct xnandps_command_format xnandps_commands[] = {
 	{NAND_CMD_READ0, NAND_CMD_READSTART, 5, XNANDPS_CMD_PHASE},
 	{NAND_CMD_RNDOUT, NAND_CMD_RNDOUTSTART, 2, XNANDPS_CMD_PHASE},
 	{NAND_CMD_READID, NAND_CMD_NONE, 1, NAND_CMD_NONE},
@@ -741,7 +741,7 @@ static void xnandps_cmd_function(struct mtd_info *mtd, unsigned int command,
 			    int column, int page_addr)
 {
 	struct nand_chip *chip = mtd->priv;
-	struct xnandps_command_format *curr_cmd = NULL;
+	const struct xnandps_command_format *curr_cmd = NULL;
 	struct xnandps_info *xnand =
 		container_of(mtd, struct xnandps_info, mtd);
 	void __iomem *cmd_addr;
-- 
1.7.1

